content/dart/concepts/queue/terms/add/add.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '.add()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'A Dart method to append or concatenate two strings.' 
+Subjects: 
+  - 'Developer Tools'
+  - 'Code Foundations'
+Tags:
```

Comment:
@ZakFarmer84  add the following in Tags:
1.  'Dart'
2.  'Programming'

because Dart is a programming language and we are working on .add() function in dart.